src/analysis/timeline/timelineCredView.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import {sum} from "d3-array";
+import sortBy from "lodash.sortby";
+import stringify from "json-stable-stringify";
+import * as NullUtil from "../../util/null";
+import {TimelineCred, type CredNode} from "./timelineCred";
+import {type Interval, weekIntervals} from "./interval";
+import {type NodeAddressT, NodeAddress} from "../../core/graph";
+
+// test intervals in timelineCredView subsset of the
+// intervals from the base TimelineCred
+export class TimelineCredView {
```

Comment:
Needs a class docstring that explains what this class is for. As an example, consider the docstrings in the `timelineCred.js` module.